shared/fs/browser/destination-picker/index.tsx

```diff
@@ -45,26 +75,31 @@ const DesktopHeaders = (props: Props) => (
         ”
       </Kb.Text>
     </Kb.Box2>
-    <Kb.Box2 direction="horizontal" fullWidth={true} centerChildren={true} style={styles.anotherHeader}>
-      <NavHeaderTitle inDestinationPicker={true} path={props.parentPath} />
-      {!!props.onNewFolder && <NewFolder onNewFolder={props.onNewFolder} />}
-    </Kb.Box2>
-  </>
-)
+  )
+}
 
 const DestinationPicker = (props: Props) => {
   FsCommon.useFsPathMetadata(props.parentPath)
   FsCommon.useFsTlfs()
   FsCommon.useFsOnlineStatus()
   return (
-    <Kb.PopupWrapper
-      onBack={props.onBack}
-      onCancel={props.onCancel}
-      customComponent={props.customComponent}
-      headerStyle={props.headerStyle}
+    <Kb.Modal
```

Comment:
maybe should be `Modal2` now?

Reply:
lets hold off on modal2 until more nav5 stuff is going

Reply:
I think I tried using that but some part didn't work out. Will try again!